docker-compose.yml

```diff
@@ -96,6 +96,7 @@ services:
       - ISSUANCE_DID_POLICY=EXISTING_OR_NEW
       - ISSUANCE_SCHEMA_POLICY=EXISTING_OR_NEW
       - ISSUANCE_CRED_DEF_POLICY=EXISTING_OR_NEW
+      - DEMO_MODE=false
```

Comment:
so does this env now stay in docker-compose and we just enable it as needed? (instead of adding it as before?)

Reply:
yes, you are correct, not sure but the default is false as well, added it there as I found it useful, but would need to set to true

Reply:
we should default this to `true`. Then in integration tests we can reset both sides very easily